app/assets/javascripts/app/discussions/discussions.module.js

```diff
@@ -41,41 +51,22 @@
       })
       .state('discussions.new.desktop', {
         views: {
-          'header@discussions.new': {
-            templateUrl: 'discussions/templates/header_desktop.html'
-          },
-          'form@discussions.new': {
+          '':{
```

Comment:
This what? They are like the other routes, if you are referring to the " " { templateUrl... } this is need because you want to render this templates inside the ui-view on discussion.html so that you can share the controller